examples/actor_spawning_and_causality.py

```diff
@@ -2,15 +2,19 @@
 
 
 def cellar_door():
-   return "Dang that's beautiful"
+    assert not tractor.is_root_process()
+    return "Dang that's beautiful"
 
 
 async def main():
     """The main ``tractor`` routine.
     """
     async with tractor.open_nursery() as n:
 
-        portal = await n.run_in_actor('some_linguist', cellar_door)
+        portal = await n.run_in_actor(
+            cellar_door,
+            name='some_linguist',
+        )
```

Comment:
I appreciate this different ordering of the arguments. It strikes my sensibilities more favorably.